app/components/InputForm.jsx

```diff
@@ -20,14 +20,16 @@ const InputForm = ({ user, tab, messages, setMessages }) => {
 			},
 			{
 				sender: 'OML Copilot',
-				message: interpretedResult.queryInterpretation,
+				message: interpretedResult.queryInterpretation !== undefined
```

Comment:
thanks for fixing this casued firestore to not save occasionaly, because null value